Kernel/Devices/KCOVInstance.cpp

```diff
@@ -37,7 +37,7 @@ ErrorOr<void> KCOVInstance::buffer_allocate(size_t buffer_size_in_entries)
     return {};
 }
 
-void KCOVInstance::buffer_add_pc(u64 pc)
+NO_SANITIZE_COVERAGE void KCOVInstance::buffer_add_pc(u64 pc)
```

Comment:
if core Kcov code is already skipped for all coverage instrumentation, does it make sense to mark this function? It's not *strictly* necessary.

Reply:
True, not required. Removed.